src/controllers/main.ts

```diff
@@ -186,3 +193,66 @@ export const getHelpController = async (
     next(error);
   }
 };
+
+/**
+ * list the pages handled with the external CMS
+ *
+ * Record of <internalSlug, externalSlug>
+ */
+const EXISTING_EXTERNAL_PAGES: Record<string, string> = {
+  accessibilite: "moncomptepro-accessibilite",
+  confidentialite: "moncomptepro-politique-de-confidentialite",
+  "conditions-generales": "moncomptepro-conditions-generales-d-utilisation",
+};
+
+export const getLegalPageController = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (!req.params.slug || !EXISTING_EXTERNAL_PAGES[req.params.slug]) {
+    return next(new NotFoundError());
+  }
+
+  const existingPage = await findPageBySlug(req.params.slug);
+  const content = {
+    body: existingPage?.body || "",
+    title: existingPage?.title || "",
+  };
+
+  // if the page is not in the database, or if it was updated more than a day ago and we want to update content automatically, fetch content
```

Comment:
these cache mecanisms should be done in https://github.com/betagouv/authentification-legal